### DISCUSSION TITLE
Can I set a different host-name for eth0 and wlan0 while using comitup?

### DISCUSSION THREAD

**Discussion by Vince-vd** [davesteele/comitup#235] [Q&A]
Hi,

First of all, thank you for creating this software, it's been so much easier to use than trying to make my own solution.

My question:
I saw somewhere that you have your own dhcp configuration file in `dns-connected.conf` and was wondering if there's any way to specify a hostname for ethernet and wlan similar to the `send host-name` argument in dhclient.conf?
My RPi will often start out connected to both wlan0 and eth0, and I would like to make sure I connect to it through the ethernet interface in that case. 

Simply disconnecting wlan0 when ethernet is connected would also be a good solution for me, is that easier potentially?

Thanks for your help!
Vincent

**Comment by davesteele** [comment]
This [issue](https://github.com/davesteele/comitup/issues/168) has come up before. It is not actively being worked.

If this is a concern for you, you could weight the ethernet route to ensure it is the one that is used.

**Reply by Vince-vd** [reply]
Thank you for the answer!
In which file should I add the weight? I'm under the impression that you don't use the standard dhcp files for comitup, or is that not right?

**Reply by davesteele** [reply]
E.g. https://unix.stackexchange.com/questions/344974/how-can-i-make-changes-to-the-network-routing-metric-permanently

It is stored in a connection file in /etc/NetworkManager/system-connections.
